gateway-ha/src/main/java/io/trino/gateway/ha/handler/QueryIdCachingProxyHandler.java

```diff
@@ -85,19 +92,20 @@ public class QueryIdCachingProxyHandler
     private final Meter requestMeter;
     private final List<String> extraWhitelistPaths;
     private final String applicationEndpoint;
+    private final Set<String> cookiePaths;
+    private final Set<String> removeCookiePaths;
+    private final DefaultSessionIdManager sessionIdManager = new DefaultSessionIdManager(new Server());
+
+    public QueryIdCachingProxyHandler(QueryHistoryManager queryHistoryManager, RoutingManager routingManager, RoutingGroupSelector routingGroupSelector, int serverApplicationPort, Meter requestMeter, List<String> extraWhitelistPaths, Set<String> cookiePaths, Set<String> removeCookiePaths)
```

Comment:
@willmostly Looks like you applied a wrong formatter on this file.

Reply:
Could you elaborate? I use IntelliJ with the [Airlift codestyle](https://github.com/airlift/codestyle) for formatting.

Reply:
I thought the codestyle was `Method declaration parameters: Chop down if long + New line after '('`, but looks like they were not set. I think https://github.com/airlift/codestyle needs an update.

Examples:
https://github.com/trinodb/trino/blob/9410a6db2d8d152658f04424777a6f4e54ce7d4b/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java#L50-L58
https://github.com/trinodb/trino/blob/9410a6db2d8d152658f04424777a6f4e54ce7d4b/plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcFileWriterFactory.java#L123-L134

---

This also doesn't look right (basically unreadable). `Binary expressions` might needs an update.
https://github.com/trinodb/trino-gateway/blob/5a659c3b2931709f28ede8a27fa8a8688a5d67ca/gateway-ha/src/main/java/io/trino/gateway/ha/handler/QueryIdCachingProxyHandler.java#L260

Reply:
https://github.com/airlift/codestyle/pull/13#issuecomment-885611067
Seems there are side effects for these rules, so they were not added. We might have to enforce it manually.

Reply:
Thanks, i didn't realize this rule wasn't in effect - I will avoid allowing IntelliJ to remove custom line breaks going forward!